src/test/java/org/openlmis/requisition/service/JasperTemplateServiceTest.java

```diff
@@ -323,7 +331,7 @@ public void mapRequestParametersToTemplateShouldReturnEmptyMapIfNoTemplateParame
   @Test
   public void mapRequestParametersToTemplateShouldReturnEmptyMapIfNoRequestParameters() {
     JasperTemplateParameter templateParameter = new JasperTemplateParameter(template, PARAM1, 
-        null, null, null, null, null, null, null, null, null);
+        null, null, null, null, null, null, null, null, null, null);
```

Comment:
@palbecki @pnaw94 I see this pull request has already been merged, but I think it's worth it to mention that even though there's the NoArgsConstructor annotation, we shouldn't use it in our code. It's there for Hibernate. I can see why the AllArgsConstructor was removed, but the NoArgs one doesn't seem to better, especially since a template parameter object should have some things set at construction time.

Ideally, we would create a constructor (or static factory method) with parameters that make sense; that seems more object-oriented.